src/client/scenes/Game.ts

```diff
@@ -139,33 +140,30 @@ export default class Game extends Phaser.Scene {
   // function to add new player to the otherPlayer group
   private handlePlayerJoined(newPlayer: IPlayer, key: string) {
     if (!this.otherPlayers) {
-      this.otherPlayers = this.physics.add.group({ classType: OtherPlayer })
+      this.otherPlayers = new Map<string, OtherPlayer>()
```

Comment:
Why check this every time instead of just initialize the Map directly once?
```ts
private otherPlayers = new Map<string, OtherPlayer>();
```
that way you also don't need to guard against the Map being nullish too.

Reply:
Done.